cpp/oneapi/dal/backend/primitives/functors.hpp

```diff
@@ -0,0 +1,104 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#pragma once
+
+#include <algorithm>
+#include <cmath>
+#include <limits>
+
+namespace oneapi::dal::backend::primitives {
+
+// Unary Functors Section
```

Comment:
Don't create obvious comments - you should have a strong value to create a comment.

Reply:
It's reasonable. I'll fix it